tapir/shifts/management/commands/apply_shift_cycle_start.py

```diff
@@ -25,7 +25,7 @@ def handle(self, *args, **options):
         if new_cycle_start_date is None:
             return
 
-        while datetime.date.today() >= new_cycle_start_date:
+        while timezone.now().date() >= new_cycle_start_date:
```

Comment:
We're missing the timezone import in this file.

Reply:
Thanks a lot for fixing it.